### ISSUE TITLE
error when running mnist_dpsgd_tutorial_keras

### ISSUE THREAD

**Issue by imaginedragontt** [tensorflow/privacy#51]
when i was running the code , there is a error as follows " module 'tensorflow.python.keras.losses' has no attribute 'CategoricalCrossentropy'" , i am wondering if it is because i have an old version of tensorflow , if it is ,please tell me which version you are using ,thx.

**Comment by npapernot** [comment]
That would explain why you are missing the loss yes. If you are using a version superior or equal to TF.1.13, feel free to re-open

**Comment by imaginedragontt** [comment]
sorry to bother but i am using tf1.5.0 , i really don't know why i have this issue

**Comment by npapernot** [comment]
TF.1.5.0 is quite old and does not have that class yet: https://github.com/tensorflow/docs/blob/r1.5/site/en/api_docs/python/index.md
